optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/SingleStatistic.java

```diff
@@ -178,6 +178,18 @@ public void readCsvStatisticFile() {
         }
     }
 
+    public void retrievePointList() {
+        if (getCsvFile().exists() && pointList == null) {
+            initPointList();
+            readCsvStatisticFile();
+        }
+    }
+
+    public void persistPointList() {
```

Comment:
2 votes on the same proposal is cheating :) I guess you wanted to use "pro" and "con" and you're +0 on it (which means you're hoping there's a better alternative):

According to apache voting guidelines:
-1 implies a (soft?) *veto* for as long as the comment after the "-1" cannot be countered (adding a comment is mandatory). The new situation is worse than the old.
+1 implies I think this is a good idea. The new situation is better than the old.
+0 (or -0) implies I don't care or this change can slip through or I prefer another alternative even though this has it's merits. The new situation isn't worse than the old.

The philopshy is that one +1 vote can get a change in, as long as there are none -1's. And also that -1's need to be explicitly dealt with (by turning them into +0 for example).